developing_packages/package_layout.rst

```diff
@@ -0,0 +1,326 @@
+.. _package_layout:
+
+Package layout
+==============
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+Available since: `1.37.0 <https://github.com/conan-io/conan/releases/tag/1.37.0>`_
```

Comment:
```suggestion
Available since: `1.37.0 <https://github.com/conan-io/conan/releases>`_
```
That link is making the ci fail, maybe we can point to the general releases link?